student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission, name)
+    @classroom = classroom
+  end
```

Comment:
- [ ] You did a great job working on this milestone! 💯 However, please make sure that you're passing the arguments to your `super` correctly:

If you look closely, your `Person` class constructor expects two named (or hashed) parameters: `name` and `parent_permission`, however, you're passing them as normal ones, please consider passing them correctly! 🤓 🔧 

```rb
  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
    super(age, parent_permission: parent_permission, name: name)
    @classroom = classroom
  end
```